### PR TITLE
[ORBIT] Add Ambassadors section

### PR DESCRIPTION
**Pull request by CharlyMartin**
**Large screens**
Screen Shot 2024-06-27 at 18 20 48

**Small screens**
Screen Shot 2024-06-27 at 18 25 42

[Notion](https://www.notion.so/filecoin/FF-V4-ORBIT-Add-Ambassadors-Section-3cd9e39ba53c43f3b941b2b56df9d7a7?pvs=4)

### CODE REVIEW

File: src/app/_components/OrbitAmbassadorCard.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import clsx from 'clsx'
+
+import { BasicCard } from '@/components/BasicCard'
+import { StaticImage, type StaticImageProps } from '@/components/StaticImage'
+
+export type OrbitAmbassadorCardProps = {
+  index: number
+  image: StaticImageProps
+  name: string
+  testimonial: Array<string> | Array<JSX.Element>
```
Comment (barbaraperic) [inline src/app/_components/OrbitAmbassadorCard.tsx:10]: should we keep using `string[]` format?

File: src/app/_components/OrbitAmbassadorCard.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  location: string
+}
+
+import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
```
Comment (barbaraperic) [inline src/app/_components/OrbitAmbassadorCard.tsx:14]: this one ended up below `OrbitAmbassadorCardProps`, should be with other import statements

File: src/app/_components/OrbitAmbassadorCard.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  name,
+  testimonial,
+  location,
+}: OrbitAmbassadorCardProps) {
+  const isImagePositionOdd = index % 2 === 1
+
+  return (
+    <article
+      className={clsx(
+        'flex flex-col gap-4 lg:flex-row lg:gap-6',
```
Comment (barbaraperic) [inline src/app/_components/OrbitAmbassadorCard.tsx:28]: should be gap-6 (24px) according to designs

Comment (mirhamasala) [reply src/app/_components/OrbitAmbassadorCard.tsx:28]: I think this better groups the cards on smaller screens. I'll talk to Filipa. 🙏🏼

**Review by barbaraperic — APPROVED** [review]
Commented on some small things, looks good 👌

**Pull request merged**